### PR TITLE
Basic description of tuples

### LINKED ISSUE #86: Add an out of date notice
The handbook currently doesn't mention things like tuples, and this makes it confusing if you learn about tuples from a different source and then try to look them up in the handbook. The handbook is also the main form of documentation in the TS website, so it's natural to expect it being up to date, which is an another reason to add a notice that it doesn't cover all the TS features.

### PR DESCRIPTION
**Pull request by sandersn**
Fixes #86.

Clone from "What's new in Typescript" into Basic Types. A more complete example should be put elsewhere, along with an explanation of destructuring. The handbook focusses almost exclusively on types and modules right now, since that's what Typescript adds. So there's not really a good place to talk about expressions right now.

### CODE REVIEW

**Comment by mhegazy** [comment]
:+1:

File: pages/Basic Types.md [hunk 1]
```diff
@@ -69,6 +69,35 @@ The second way uses a generic array type, `Array<elemType>`:
 var list: Array<number> = [1, 2, 3];
 ```
 
+# Tuple
+
+Tuple types allow you to express an array where the type of a fixed number of elements is known, but need not be the same.
+For example, you may want to represent an array with a `string` at position 0 and a `number` at position 1:
```
Comment (DanielRosenwasser) [inline pages/Basic Types.md]: represent a value as a pair of a `string` and a `number`

File: pages/Basic Types.md [hunk 2]
```diff
@@ -69,6 +69,35 @@ The second way uses a generic array type, `Array<elemType>`:
...
+```
+
+When accessing an element with a known index, the correct type is retrieved:
+
+```TypeScript
+console.log(x[0].substr(1)); // OK
+console.log(x[1].substr(1)); // Error, 'number' does not have 'substr'
+```
+
+When accessing an element outside the set of known indices, a union type is used instead:
```
Comment (DanielRosenwasser) [inline pages/Basic Types.md]: We'll need to eventually link to union types

File: pages/Basic Types.md (hunk 1 above)
Comment (sandersn) [inline pages/Basic Types.md]: done

File: pages/Basic Types.md (hunk 2 above)
Comment (sandersn) [inline pages/Basic Types.md]: done, although I'm not even sure of the value of covering the error case -- I just copied it from the release notes. We kind of need a tutorial-style handbook *and* a set of reference pages that explicate the spec (@weswigham's tutorial on migrating to ES2015 modules is another good candidate for reference documentation). I'll leave it in until we make a reference section, though.

Comment (DanielRosenwasser) [inline pages/Basic Types.md]: What error case?

Comment (sandersn) [inline pages/Basic Types.md]: I was thinking of a runtime error, which we do not catch at compile time..

```ts
var x: [string, number] = ['hello', 10];
console.log(x[5].toString()); 
// would have been OK if we assigned to x[5] earlier, but we didn't in this case.
```

Telling people that they can add additional entries and properties to a tuple is not the most basic use of them -- it falls out from their array and object inheritance.

**Comment by DanielRosenwasser** [comment]
:+1:

**Pull request merged**